### DISCUSSION TITLE
Simple way to create a formula for a script?

### DISCUSSION THREAD

**Discussion by danielfalbo** [Homebrew/discussions#1065] [Getting started] [answered]
Hi! So, I have a simple python script called [now](https://github.com/danielfalbo/dotfiles/blob/master/bin/now), and I would like to write and push to my GitHub account a brew formula so that after `brew install danielfalbo/formulae/now`, the script should be added to my `PATH` and I should be able to call it from anywhere in my terminal as just `$ now`. Does anyone know how to do this without complicated stuff? I've seen a couple of brew formulas and I'm not sure I fully understand how they work but I've seen a lot of Makefiles.... do I have to dive deeper or is there a simpler way to do this?

**Comment by SMillerDev** [comment, marked as answer]
You don't need makefiles per se, since you can just use `bin.install "filename"` to have the same effect.
You will need to use https://docs.brew.sh/Formula-Cookbook#specifying-the-download-strategy-explicitly to specify a non-extracting download strategy.

**Reply by danielfalbo** [reply]
Thank you! Exactly what I was looking for :)
